Makefile

```diff
@@ -38,33 +37,33 @@ PROTOC_VERSION    ?= 3.4.0
 # Referenced by github.com/improbable-eng/thanos/blob/master/docs/getting_started.md#prometheus
 
 # Limitied prom version, because testing was not possibe. This should fix it: https://github.com/improbable-eng/thanos/issues/758
-PROM_VERSIONS ?=v2.4.3 v2.5.0
+PROM_VERSIONS           ?=v2.4.3 v2.5.0
 ALERTMANAGER_VERSION    ?=v0.15.2
 MINIO_SERVER_VERSION    ?=RELEASE.2018-10-06T00-15-16Z
 
 # fetch_go_bin_version downloads (go gets) the binary from specific version and installs it in $(BIN_DIR)/<bin>-<version>
 # arguments:
-# $(1): Install path. (e.g github.com/golang/dep/cmd/dep)
+# $(1): Install path. (e.g github.com/campoy/embedmd)
 # $(2): Tag or revision for checkout.
 define fetch_go_bin_version
 	@mkdir -p $(BIN_DIR)
 
 	@echo ">> fetching $(1)@$(2) revision/version"
 	@if [ ! -d '$(TMP_GOPATH)/src/$(1)' ]; then \
-    GOPATH='$(TMP_GOPATH)' go get -d -u '$(1)'; \
+    GOPATH='$(TMP_GOPATH)' GO111MODULE='off' go get -d -u '$(1)/...'; \
   else \
     CDPATH='' cd -- '$(TMP_GOPATH)/src/$(1)' && git fetch; \
   fi
 	@CDPATH='' cd -- '$(TMP_GOPATH)/src/$(1)' && git checkout -f -q '$(2)'
 	@echo ">> installing $(1)@$(2)"
-	@GOBIN='$(TMP_GOPATH)/bin' GOPATH='$(TMP_GOPATH)' go install '$(1)'
+	@GOBIN='$(TMP_GOPATH)/bin' GOPATH='$(TMP_GOPATH)' GO111MODULE='off' go install '$(1)'
 	@mv -- '$(TMP_GOPATH)/bin/$(shell basename $(1))' '$(BIN_DIR)/$(shell basename $(1))-$(2)'
 	@echo ">> produced $(BIN_DIR)/$(shell basename $(1))-$(2)"
 
 endef
 
 .PHONY: all
-all: deps format build
+all: format build
```

Comment:
can we add `mod-tidy` here so that it runs tidy everytime we run `make`?  Will save me from having to remember it :)

Reply:
Also add `bzr-check` for local ... otherwise `go build` and `go mod` will fail and users will be unaware ... if they do not have `bzr` please link them to the install location :+1:

Reply:
Done